src/components/Points/ActivityPointsFeed.tsx

```diff
@@ -2,44 +2,82 @@ import { GLOBAL_CAMPAIGN_IDS } from '@/config/constants'
 import { Campaign } from '@/hooks/useCampaigns'
 import { useChainId } from '@/hooks/useChainId'
 import { useOwnCampaignRank } from '@/hooks/useLeaderboard'
-import { formatDatetime } from '@/utils/date'
+import { formatDate } from '@/utils/date'
 import { Divider, Skeleton, Stack, Typography } from '@mui/material'
 import Box from '@mui/material/Box'
-import { useEffect, useMemo, useState } from 'react'
+import { ReactNode, useEffect, useMemo, useState } from 'react'
 import PointsCounter from '../PointsCounter'
 import Barcode from '@/public/images/horizontal_barcode.svg'
 import css from './styles.module.css'
+import { useLatestCampaignUpdate } from '@/hooks/useLatestCampaignUpdate'
+
+const BorderedBox = ({ children }: { children: ReactNode }) => {
+  return (
+    <Box
+      sx={{
+        border: ({ palette }) => `1px solid ${palette.border.light}`,
+        borderRadius: '6px',
+        p: '24px 32px',
+        display: 'flex',
+        flexDirection: 'column',
+        gap: 2,
+        position: 'relative',
+      }}
+    >
+      {children}
+    </Box>
+  )
+}
+
 const HiddenValue = () => (
   <Typography minWidth="80px">
     <Skeleton />
   </Typography>
 )
 
+const DataWrapper = ({ children }: { children: ReactNode }) => {
+  return (
+    <Stack
+      direction={{ xs: 'column', lg: 'row' }}
+      justifyContent="space-between"
+      alignItems={{ xs: 'start', lg: 'center' }}
+      spacing={2}
+    >
+      {children}
+    </Stack>
+  )
+}
+
 export const ActivityPointsFeed = ({ campaign }: { campaign?: Campaign }) => {
   const { data: ownEntry, isLoading } = useOwnCampaignRank(campaign?.resourceId)
+  const { data: latestUpdate, isLoading: isLatestUpdateLoading } = useLatestCampaignUpdate(campaign?.resourceId)
 
   const data = useMemo(() => {
     return {
-      activityPoints: ownEntry?.totalPoints ?? 0,
-      boostedPoints: ownEntry ? ownEntry.totalBoostedPoints - ownEntry.totalPoints : 0,
-      totalPoints: ownEntry?.totalBoostedPoints ?? 0,
+      activityPoints: latestUpdate?.totalPoints ?? 0,
+      boostedPoints: latestUpdate ? latestUpdate.totalBoostedPoints - latestUpdate.totalPoints : 0,
+      totalPoints: latestUpdate?.totalBoostedPoints ?? 0,
+      overallPoints: ownEntry?.totalBoostedPoints ?? 0,
     }
-  }, [ownEntry])
+  }, [latestUpdate, ownEntry])
 
   const [showBoostPoints, setShowBoostPoints] = useState(false)
   const [showTotalPoints, setShowTotalPoints] = useState(false)
+  const [showOverallPoints, setShowOverallPoints] = useState(false)
 
   useEffect(() => {
-    if (ownEntry !== undefined || !isLoading) {
+    if (ownEntry !== undefined || !isLoading || !isLatestUpdateLoading) {
```

Comment:
Should this be `!isLoading && !isLatestUpdateLoading` to only show the points if all loading has finished? It might be easier to read with an early return i.e. `if (ownEntry === undefined || isLoading || isLatestUpdateLoading) return`